City-of-Helsinki/drupal-helfi-kasvatus-koulutus
# UHF-3453: option to hide description on TPR Units

How to test:

1. Set up KASKO site
2. Checkout this branch: `git co UHF-3453_tpr_unit_hide_description`
3. Checkout correct branch of the TPR module: `composer require drupal/helfi_tpr:dev-UHF-3453_tpr_unit_hide_description`
4. Run database changes (from the TPR module): `make drush-updb`
5. Import config (from this branch): `make drush-cim`
6. Import some TPR Units (try to import enough to get some KASKO specific ones)
  6.1 `make shell`
  6.2 `MIGRATE_LIMIT=1000 drush mim tpr_unit`
7. Go to the list of TPR Units and search for "lukio" and edit one of them (for example "Otaniemen lukio", ID 61450 is good)
8. Below the description, there is a checkbox "Hide description". Select it and save the entity.
9. View the entity and see that the description is not visible

See also https://github.com/City-of-Helsinki/drupal-module-helfi-tpr/pull/80
